runtime/compiler/runtime/JITClientSession.cpp

```diff
@@ -501,14 +507,6 @@ ClientSessionData::clearCaches()
 
    _classChainDataMap.clear();
```

Comment:
`clearCaches()` must clear the CHTable as well.

Reply:
I clear the CHTable in `ClientSessionData` destructor, since it was previously cleared there.
I will move it to `clearCaches()` if you'd prefer that.

Reply:
In case some message gets lost we have to clear all caches and start all over because that message may have carried important class unload and CHTable modification that needs to be applied. The client session data is not destroyed, just the caches are cleared, including the CHTable.